molten-test/src/main/java/com/hotels/molten/test/mockito/ReactiveAnswer.java

```diff
@@ -17,28 +17,34 @@
 package com.hotels.molten.test.mockito;
 
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.withSettings;
 
-import org.mockito.internal.stubbing.defaultanswers.ReturnsEmptyValues;
+import java.util.Map;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
 import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
 /**
- * Mockito default return values for Reactor reactive types.
+ * Mockito answer to return default values for Reactor reactive types.
  */
-public class ReactiveAnswer extends ReturnsEmptyValues {
+@RequiredArgsConstructor
+public class ReactiveAnswer implements Answer<Object> {
+    private final Map<Class<?>, Object> reactiveDefaultAnswers = Map.of(
+        Mono.class, Mono.empty(),
```

Comment:
Should defer creation of Mono/Flux

Reply:
I did not deferred them as they are returning the singleton `MonoEmpty/FluxEmpty#INSTANCE`, and no `onAssembly()` is called.

Reply:
Right. Thanks.